src/applications/mhv-landing-page/selectors/hasMhvBasicAccount.js

```diff
@@ -0,0 +1,12 @@
+import { isLOA1 } from '~/platform/user/selectors';
+import { hasMhvAccount } from './hasMhvAccount';
+
+/**
+ * Selects the hasMHVBasicAccount state based on user being LOA1 and having an MHV account.
+ * @param {Object} state The current application state.
+ * @returns {boolean} The hasMHVBasicAccount state.
+ */
+
+export const hasMhvBasicAccount = state => {
+  return isLOA1(state) && hasMhvAccount(state);
```

Comment:
Did you look into `state.mhvAccount.accountLevel`? I see that in some [vets-website code](https://github.com/department-of-veterans-affairs/vets-website/blob/77357-ver-2/src/platform/user/tests/profile/reducers/index.unit.spec.js).

I think the `isLOA1` only refers to their current login on VA.gov, e.g. idme or login-dot-gov. I think someone could have an idme/logingov account that is LOA3 while their account on the MHV National Portal is Basic (and therefore unable to access most MHV data)

Reply:
Re-read the ticket https://github.com/department-of-veterans-affairs/va.gov-team/issues/77357, and the following makes it sound like we need to check if the veteran is currently logged in with an MHV login:

> As a Veteran signed-in using a non-LOA3 My HealtheVet basic credential…

That's a different from my original read of it, and now I think we need to look at the `profile.login.serviceName` against `CSP_IDS.MHV` (or similar)

Reply:
Sure thing, I will try that out. I didn't know what "basic credentials" was or what these login "services" are. Didn't know "MHV login" was a thing. I just saw LOA1 and mhv account but a basic account is something new to me.